src/electron/views/automated-checks/automated-checks-view.tsx

```diff
@@ -6,6 +6,7 @@ import { GetCardViewData } from 'common/rule-based-view-model-provider';
 import { CardSelectionStoreData } from 'common/types/store-data/card-selection-store-data';
```

Comment:
Changes in this file are hardly ideal but this class is in a strong need of a re-factor that I did not want to do as a part of this PR.